etc/deploy/clear_kops_resources_in_aws.sh

```diff
@@ -5,7 +5,7 @@
 # though, so that if we're ever trying to clean up after a kops cluster again, and we need to
 # figure out what to delete, we'll have a place to start.
 
-region=us-east-1
+region="${REGION:-us-west-1}"
```

Comment:
Since the script already sets `AWS_REGION`, I think it might be simpler/safer to replace `${region}` with `${AWS_REGION}` everywhere and delete this line